htlcswitch/quiescer.go

```diff
@@ -398,6 +426,10 @@ func (q *QuiescerLive) sendOwedStfu(numPendingLocalUpdates uint64) error {
 		// state. If so, we will try to resolve any outstanding
 		// StfuReqs.
 		q.tryResolveStfuReq()
+
+		if q.isQuiescent() {
```

Comment:
non-blocking / can be done in a followup if we feel strongly: we're doubling up on `q.isQuiescent` checks here and in `tryResolveStfuReq` - could be combined into one function that handles all our tasks once we're quiescent.